android/app/build.gradle

```diff
@@ -138,7 +138,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "org.pathcheck.covidsafepaths.bt"
+        applicationId "org.pathcheck.gr.bt"
```

Comment:
This change shouldn't be committed, we use the general app ID `org.pathcheck.covidsafepaths.bt` for debugging and the specific app IDs are changed during the CI process